mix.exs

```diff
@@ -42,8 +42,8 @@ defmodule ReqBigQuery.MixProject do
   defp deps do
     [
       {:decimal, "~> 2.0"},
-      {:req, "~> 0.3.5 or ~> 0.4.0"},
-      {:goth, "~> 1.3.0"},
+      {:req, ">= 0.4.0"},
+      {:goth, ">= 1.3.0"},
```

Comment:
```suggestion
      {:req, "~> 0.3.5 or ~> 0.4"},
      {:goth, "~> 1.3"},
```

Reply:
@josevalim thanks for the review, we are trying to loosen deps up so we can use req 0.5.0 and goth 1.4.2 would this be acceptable? 

```suggestion
      {:req, "~> 0.5.0"},
      {:goth, "~> 1.4.0"},
```

Reply:
The one I proposed will do that, but not allow different major versions.